spec.md

```diff
@@ -722,7 +770,7 @@ the current value of the element, writes are not visible until after a positive
 edges of the register's clock port.
 
 The clock signal for a register must be of type `Clock`{.firrtl}.  The type of a
-register must be a passive type (see [@sec:passive-types]).
+register must be a passive type (see [@sec:passive-types]) and may not be const.
```

Comment:
```suggestion
register must be a passive type (see [@sec:passive-types]) and may not be `const`{.firrtl}.
```

Maybe? I'm not sure the rules when we use these modifiers or not

Reply:
Me either, but for lack of a better idea I think they should be used on all FIRRTL language snippets (including variable references), but certainly involving keywords.

I don't think there's consistent use, although I added it to a few places recently.   Any consistent rule is probably an improvement.